wiki/floodgate/index.md

```diff
@@ -1,15 +1,13 @@
 ---
-title: Floodgate
+title: Overview
 permalink: /floodgate/
 description: 'Floodgate is a hybrid mode plugin that allows Minecraft: Bedrock Accounts to join Minecraft: Java Edition servers without needing a Minecraft: Java Edition account.'
 ---
 
-# Floodgate
-## What is Floodgate? {#what-is-floodgate}
+# Overview
 Floodgate is a hybrid mode plugin that allows Minecraft: Bedrock Accounts to join Minecraft: Java Edition servers without needing a Minecraft: Java Edition account. 
 This is something you install in addition to Geyser. Unlike Geyser, Floodgate can only be installed as a plugin on Spigot (including Paper and forks), Bungeecord, Velocity, and as a mod on Fabric servers, and can only be utilized on servers that have it installed.
 
-## What has Floodgate 2.0 to offer? {#what-has-floodgate-20-to-offer}
-Floodgate 2.0 is a rewrite of Floodgate 1.0, with various bugfixes and improvements. It also introduces new features: being able to [see Bedrock player skins on Java edition](/wiki/floodgate/features#what-is-skin-uploading),
+Now, the version of Floodgate is 2.0. Floodgate 2.0 is a rewrite of Floodgate 1.0, with various bugfixes and improvements. It also introduces new features: being able to [see Bedrock player skins on Java edition](/wiki/floodgate/features#what-is-skin-uploading),
```

Comment:
You can drop the mention/comparison of floodgate entirely now, it's been out for a while now